package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "axios": "^0.24.0"
+  }
+}
```

Comment:
It looks like the `axios` dependency got added despite you not using it in this project. This is an easy mistake to make! Just make sure you're careful to only include files you actually want when you're committing to avoid bloating your repository with unneeded dependencies.